### ISSUE TITLE
Missing files: Kyria HiPro Case Plate, No Puck

### ISSUE THREAD

**Issue by LuisMorenoSerrano** [splitkb/kyria#10]
Latest versions of **Kyria HiPro Case Plate** (_0.1mm Kerf_ and _No Kerf_) are **Puck versions** both. I would like to have **No Puck** versions too, but I have no software for making this kind of change.

Thanks a lot for your impressive work, regards.

**Comment by bbaserdem** [comment]
@LuisMorenoSerrano you can check commit history; the previous files are the no-puck versions.

**Comment by LuisMorenoSerrano** [comment]
You are right @silverbluep, thanks for the advice. I have downloaded deleted file (no puck version) but, apart of new holes, it is not exactly the same as latest version... Finally I have manually edited latest version and I removed the new four circles, so now I have a no puck version :)

Regards
